cli/src/wallet.rs

```diff
@@ -179,6 +184,41 @@ impl WalletSubCommands for App<'_, '_> {
                         "From (base) key, [default: cli config keypair]. "),
                 ),
         )
+            .subcommand(
+                SubCommand::with_name("find-program-derived-address")
+                    .about("Generate a program derived account address with a seed")
+                    .arg(
+                        Arg::with_name("program_id")
+                                .index(1)
+                                .value_name("PROGRAM_ID")
+                                .takes_value(true)
+                                .required(true)
+                                .help(
+                                    "The program_id that the address will ultimately be used for, \n\
+                                    or one of NONCE, STAKE, and VOTE keywords",
+                                ),
+                        )
+                    .arg(
+                        Arg::with_name("seeds")
+                            .min_values(0)
+                            .value_name("SEED")
+                            .takes_value(true)
+                            .validator(is_structured_seed)
+                            .help(
+                                "The seeds. \n\
+                                Each one must match the pattern PREFIX:VALUE. \n\
+                                PREFIX can be one of [string, pubkey, hex, u8] \n\
+                                or matches the pattern [u,i][16,32,64,128][le,be] (for example, u64le) \n\
```

Comment:
Just nitpicking, but let's do "or matches the patern [u,i][16,32,64,128][le,be] (for example u64le) for number values" and then remove lines 212-214 to minimize the help message a little bit.